apps/dashboard/src/routes/org/[slug]/community/ask/+page.svelte

```diff
@@ -10,15 +10,29 @@
   import TextEditor from '$lib/components/TextEditor/index.svelte';
   import TextField from '$lib/components/Form/TextField.svelte';
   import { profile } from '$lib/utils/store/user';
+  import { fetchCourses } from '$lib/components/Courses/api';
+  import { onMount } from 'svelte';
+
+  // export let isLMS = false;
```

Comment:
remove

Reply:
i can't remove the fetchCourse, remember that's where i'm getting all the course titles to display on the dropdown

Reply:
This comment has nothing to do with fetchCourses

Reply:
oh, yeah, i removed it already, you highlighted line 13 - 16 so i had to say something there.